cmd/rpcdaemon/commands/zkevm_api.go

```diff
@@ -1067,7 +1149,7 @@ func (api *ZkEvmAPIImpl) GetProverInput(ctx context.Context, batchNumber uint64,
 		return nil, err
 	}
 
-	oldAccInputHash, err := api.l1Syncer.GetOldAccInputHash(ctx, &api.config.AddressRollup, api.config.L1RollupId, batchNumber)
+	oldAccInputHash, err := api.getAccInputHash(ctx, hDb, batchNumber)
```

Comment:
as this is "old" acc input hash, should it be for the previous batch/sequence?

Reply:
not sure about this one, maybe @kstoykov knows if we need the current batch accInputHash here as well, or it is the previous sequence's one?

Reply:
We could compare a local node with a legacy node for one of the integration networks and check if they agree?  Would be good to nail it first time if we can 💪

Reply:
Spoke with Carlos and he said this should be for previous batch. I made the change